clock.rb

```diff
@@ -18,4 +20,10 @@ module Clockwork
   end
 
   every(3.hours, 'scrapping.start')
+
+  Clockwork.manager = DatabaseEvents::Manager.new
+
```

Comment:
can you also provide an example, how it will trigger handler? Or it will be done automatically?

Reply:
Basically. Sync_database events will go to DB and fetch new data from models (we specify the time how often it checks in every field. Then actions will be executed dependent on the model frequency. Eg. The image below. "2 event" have a lower frequency than "fafa" event. So it will be executed more often.

![image](https://user-images.githubusercontent.com/26395360/68796264-8f3bc300-0620-11ea-8139-abab0ae38817.png)